### PR TITLE
Malteform-signup

### PR DESCRIPTION
**Pull request by evelinabe**
The assignment was to make a business one-pager with a hero image or video, a form and an FAQ accordion. I chose to do my one-pager for my father-in-law's site where he shows his pottery. 

**The plan**

I did a quick sketch manually and pinpointed focus-points. In my last project I got lost in the details so it was important for me this time to find a way to keep the bigger picture in mind (so that "has to have" trumps "nice to have" for sure).

I used three different video sizes for my hero video to make the site responsive for different screens. The videos that does not show are set to <display:none> in media queries. The middle screen size has a different video than the others to make the change visible. 

I added a sticky navbar with the help of Javascript. The navbar is below the video, and sticks to the top on scroll.

The main content has a form to book pottery classes for a group of friends,  and a FAQ-accordion. I used flexbox to align all content. The accordion has icons that indicates if the items is collapsed or not. It has a small transitioning on collapse. 

The page also has a footer. Small glimpses of fun are animation on hover on the navbar links, a favicon and an arrow down mouse cursor on the accordion header. 

With more time I would have added a text container in the beginning of the main content, as well as a button on the side of the content to click back up on the page.

### CODE REVIEW

**Review by bealun — APPROVED** [review]
What a fantastic business site you've built, bravo! 🌟 It feel like you've thought of everything from the small details to the bigger picture stuff too. I've left some feedback and want you to keep pushing yourself since you're doing so good. Try to push code a bit more often, say 3 to 4 times per day, and with informative commit messages and keep maintaining your code structure - this will benefit you in the long run!
One thing that stood out was the accordion that doesn't work the way we'd like and you didn't mention anything about it - which it's totally fine but check the comments and just get back to me when you can so I know if everything worked or not. You've really impressed me and done a phenomenal job, remember that! 👋 BRAVO!

**Comment by evelinabe** [comment]
Doing a happy dance :-)

I actually didn't think about the clicking on only a small part of the accordion as a bug, strange enough. Followed your suggestions and now it works, yey! Thanks for pointing that out!

In my bug-report in the README-file I wrote about the issue with the icons. The problem is that the "advanced "loop @EmOlsz provided for the accordion works differently from my try at JS with the icon. The loop closes the accordion when clicking on another accordion-header, but the icon toggles on click only. That's now changed, so the icon has the same loop and works as you want it to.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+                <p>This browser does not support video</p>
+            </video>
+          </div>
+
+          <div class="hero-video-large">
+            <video muted autoplay loop>
+                <source src="./videos/Potters_1280x720.mp4" type="video/mp4">
+                  <p>This browser does not support video</p>
+            </video>
+        </div>
```
[the remark this answers is not in the document]
Comment (evelinabe) [reply index.html:62]: It was a general requirement when choosing the video for a hero header, though?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                        value="upTo10Value"
+                    />
+                </div>
+                <div id="checkbox-group" class="form-group">
+                    <label for="terms">Jag godkänner köpvillkoren.</label>
+                    <input
+                        id="terms"
+                        type="checkbox"
+                        name="termsValue"
+                    />
```
[the remark this answers is not in the document]
Comment (evelinabe) [reply index.html:149]: Actually I did a lot of thinking about it and decided against it. But that's a longer discussion over real coffe :-D

File: index.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                <h3 id="accordion-header-1" class="accordion-header">
+                    <span>Hur många kan delta?</span>  
+                </h3>
+                <div class="accordion-textbox">
+                    <p class="accordion-text">
+                        Grupper på upp till 10 personer är välkomna.
+                        Det går också bra att komma på egen hand för 
+                        dig som vill lära dig mer.
+                    </p>
+                </div>
```
[the remark this answers is not in the document]
Comment (evelinabe) [reply index.html:180]: Worked, updated code's pushed.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+// Icon toggle function
+
+const accordionIcon = document.querySelectorAll(".accordion-header");
+
+accordionIcon.forEach(accordionIcon => {
+    accordionIcon.addEventListener("click", event => {
+        accordionIcon.classList.toggle("active");
+    });
+});
```
[the remark this answers is not in the document]
Comment (evelinabe) [reply script.js:26]: Wrote about this in the README. It's now fixed using a similar loop as Max.

**Pull request closed without merging**